### PR TITLE
Update network options.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Updated the network options and included a minor troubleshooting update.

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -744,14 +744,12 @@ Possible values are:
 
 | Network   | Chain   | Type        | Description                                                         |
 |-----------|---------|-------------|---------------------------------------------------------------------|
+| `mainnet` | Eth 2.0 | Production  | Main network.                                                       |
```
Comment (ajsutton) [inline docs/Reference/CLI/CLI-Syntax.md:747]: The `pyrmont` network has been added as well.   It's another Multi-client testnet using the v1.0 spec and very similar to the Mainnet configuration.

File: docs/Reference/CLI/Subcommands/Admin.md [hunk 2]
```diff
@@ -269,14 +269,12 @@ Possible values are:
...
 | Network   | Chain   | Type        | Description                                                         |
 |-----------|---------|-------------|---------------------------------------------------------------------|
+| `mainnet` | Eth 2.0 | Production  | Main network.                                                       |
 | `altona`  | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
-| `mainnet` | Eth 2.0 | Production  | Intended main network that targets future mainnet configuration.    |
 | `medalla` | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
 | `minimal` | Eth 2.0 | Test        | Used for local testing and development networks.                    |
-| `spadina` | Eth 2.0 | Test        | Short-lived multi-client testnet maintained by the Goerli testnet initiative.   |
 | `topaz`   | Eth 2.0 | Test        | Single-client testnet maintained by the Prysmatic Labs.             |
 | `witti`   | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
```
Comment (ajsutton) [inline docs/Reference/CLI/Subcommands/Admin.md:277]: Topaz and witti have also been removed.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Syntax.md:747]: Done

File: docs/Reference/CLI/Subcommands/Admin.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/CLI/Subcommands/Admin.md:277]: Done

**Pull request merged**